partcad/src/partcad/project_factory_git.py

```
@@ -38,12 +39,32 @@ def __init__(self):
         self.import_revision = self.config_obj.get("revision")
         self.import_rel_path = self.config_obj.get("relPath")
 
+        self._apply_import_overrides()
+
+    def _apply_import_overrides(self):
```

Ruff (0.8.2)

Missing return type annotation for private function `_apply_import_overrides`

Add return type annotation: `None`

(ANN202)